parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -393,6 +395,11 @@ public ReadBuilder recordsPerBatch(int numRowsPerBatch) {
       return this;
     }
 
+    public ReadBuilder withNameMapping(NameMapping newNameMapping) {
```

Comment:
This is called `nameMapping` in the Avro API. I think I prefer this name, but we should be consistent and either change that one or use the same name here.

Reply:
Will change Avro side to use this one as well.